src/Templates/MLDotNet/Microsoft.VisualStudio.Templates.MLDotNet/Content/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,87 @@
+/* This template shows the building blocks for training a machine learning model with ML.NET (https://aka.ms/mlnet).
+ * This model predicts whether a sentence has a positive or negative sentiment. It is based on a sample that can be 
+ * found at https://aka.ms/mlnetsentimentanalysis, which provides a more detailed introduction to ML.NET and the scenario. */
+
+using System;
+using System.IO;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Transforms.Text;
+
+namespace $safeprojectname$
+{
+    class Program
+    {
+        static void Main()
+        {
+            // 1. Build an ML.NET pipeline for training a sentiment analysis model.
+            Console.WriteLine("Training a model for Sentiment Analysis using ML.NET");
+            var mlContext = new MLContext();
+            // 1a. Load the training data using a TextLoader.
+            var reader = 
+                new TextLoader(
+                    mlContext,
+                    new TextLoader.Arguments()
+                    {
+                        Separator = "tab",
+                        HasHeader = true,
+                        Column = new[]
+                        {
+                            new TextLoader.Column("Label", DataKind.Bool, 0),
+                            new TextLoader.Column("Text", DataKind.Text, 1)
+                        }
+                    });
+
+            IDataView trainingDataView = reader.Read(new MultiFileSource(@"Data\wikipedia-detox-250-line-data.tsv"));
```

Comment:
You can drop the `new MultiFileSource`. Read has an overload that just takes a path.

Reply:
Thanks! Updated